content/applications/essentials/export_import_data.rst

```diff
@@ -476,3 +479,39 @@ The two files produced are ready to be imported in Odoo without any modification
 imported these two :abbr:`CSV (Comma-separated Values)` files, there are four contacts and three
 companies (the first two contacts are linked to the first company). Keep in mind to first import
 the companies, and then the people.
+
+Update data in Odoo
+===================
+
+Existing data can be updated in bulk through a data import, as long as the :ref:`External ID
+<essentials/external-id>` remains consistent.
+
+Prepare data export
+-------------------
+
+To update data through an import, first navigate to the data to be updated, and select the |list| to
+activate list view. On the far left side of the list, tick the checkbox for any record to be
+updated. Then, click |actions|, and select :icon:`fa-upload` :guilabel:`Export` from the drop-down
+menu.
+
+On the resulting :guilabel:`Export Data` pop-up window, tick the checkbox labeled :guilabel:`I want
+to update data (import-compatible export)`. This automatically includes the External ID in the
+export, and limits the :guilabel:`Fields to export` list to **only** include fields that are able to
+be imported. Select the required fields to be included in the export using the :ref:`options
+<export-data>` on the pop-up window, then click :guilabel:`Export`.
```

Comment:
When you say "This automatically includes the External ID in the export, and limits the 'Fields to export' list to **only** include fields that are able to be imported..." ... are you saying that the External ID will still be exported *even though it does NOT appear* on the 'Fields to export' list that populates when the checkbox for 'I want to update data (...)' is ticked? If so, a bit of clarification on that would be helpful.

Reply:
Correct, it does not appear in the list on the pop-up, but it _is_ included in the exported file. I will clarify.